src/types/jobDescription.ts

```diff
@@ -0,0 +1,356 @@
+                FROM QSYS2.JOB_DESCRIPTION_INFO
+                where JOB_DESCRIPTION_LIBRARY = '${this.library}' AND JOB_DESCRIPTION = '${this.name}' LIMIT 1`].join(` `));
+
+            if (jobDescriptionInfo && jobDescriptionInfo.length > 0) {
+                const resultJobDescription: JobDescriptionInfo[] = jobDescriptionInfo.map(row => ({
+                    jobDescriptionLibrary: String(row.jobDescriptionLibrary),
+                    jobDescription: String(row.jobDescription),
+                    authorizationName: String(row.authorizationName),
+                    jobDate: String(row.jobDate),
+                    accountingCode: String(row.accountingCode),
+                    routingData: String(row.routingData),
+                    requestData: String(row.requestData),
+                    libraryListCount: Number(row.libraryListCount),
+                    libraryList: String(row.libraryList),
+                    jobSwitches: String(row.jobSwitches),
+                    textDescription: String(row.textDescription),
+                    jobQueueLibrary: String(row.jobQueueLibrary),
+                    jobQueue: String(row.jobQueue),
+                    jobQueuePriority: Number(row.jobQueuePriority),
+                    holdOnJobQueue: String(row.holdOnJobQueue),
+                    outputQueueLibrary: String(row.outputQueueLibrary),
+                    outputQueue: String(row.outputQueue),
+                    outputQueuePriority: Number(row.outputQueuePriority),
+                    spooledFileAction: String(row.spooledFileAction),
+                    printerDevice: String(row.printerDevice),
+                    printText: String(row.printText),
+                    jobMessageQueueMaximumSize: Number(row.jobMessageQueueMaximumSize),
+                    jobMessageQueueFullAction: String(row.jobMessageQueueFullAction),
+                    syntaxCheckSeverity: String(row.syntaxCheckSeverity),
+                    jobEndSeverity: Number(row.jobEndSeverity),
+                    joblogOutput: String(row.joblogOutput),
+                    inquiryMessageReply: String(row.inquiryMessageReply),
+                    messageLoggingLevel: Number(row.messageLoggingLevel),
+                    messageLoggingSeverity: Number(row.messageLoggingSeverity),
+                    messageLoggingText: String(row.messageLoggingText),
+                    logClProgramCommands: String(row.logClProgramCommands),
+                    deviceRecoveryAction: String(row.deviceRecoveryAction),
+                    timeSliceEndPool: String(row.timeSliceEndPool),
+                    allowMultipleThreads: String(row.allowMultipleThreads),
+                    workloadGroup: String(row.workloadGroup),
+                    aspgrp: String(row.aspgrp),
+                    ddmConversation: String(row.ddmConversation)
+                }));
+                this.jobDescriptionInfo = resultJobDescription[0];
```

Comment:
You can safely assign the first result of your map with this change since there's only one result at this point.
```suggestion
                [this.jobDescriptionInfo] = jobDescriptionInfo.map(row => ({
                    jobDescriptionLibrary: String(row.jobDescriptionLibrary),
                    jobDescription: String(row.jobDescription),
                    authorizationName: String(row.authorizationName),
                    jobDate: String(row.jobDate),
                    accountingCode: String(row.accountingCode),
                    routingData: String(row.routingData),
                    requestData: String(row.requestData),
                    libraryListCount: Number(row.libraryListCount),
                    libraryList: String(row.libraryList),
                    jobSwitches: String(row.jobSwitches),
                    textDescription: String(row.textDescription),
                    jobQueueLibrary: String(row.jobQueueLibrary),
                    jobQueue: String(row.jobQueue),
                    jobQueuePriority: Number(row.jobQueuePriority),
                    holdOnJobQueue: String(row.holdOnJobQueue),
                    outputQueueLibrary: String(row.outputQueueLibrary),
                    outputQueue: String(row.outputQueue),
                    outputQueuePriority: Number(row.outputQueuePriority),
                    spooledFileAction: String(row.spooledFileAction),
                    printerDevice: String(row.printerDevice),
                    printText: String(row.printText),
                    jobMessageQueueMaximumSize: Number(row.jobMessageQueueMaximumSize),
                    jobMessageQueueFullAction: String(row.jobMessageQueueFullAction),
                    syntaxCheckSeverity: String(row.syntaxCheckSeverity),
                    jobEndSeverity: Number(row.jobEndSeverity),
                    joblogOutput: String(row.joblogOutput),
                    inquiryMessageReply: String(row.inquiryMessageReply),
                    messageLoggingLevel: Number(row.messageLoggingLevel),
                    messageLoggingSeverity: Number(row.messageLoggingSeverity),
                    messageLoggingText: String(row.messageLoggingText),
                    logClProgramCommands: String(row.logClProgramCommands),
                    deviceRecoveryAction: String(row.deviceRecoveryAction),
                    timeSliceEndPool: String(row.timeSliceEndPool),
                    allowMultipleThreads: String(row.allowMultipleThreads),
                    workloadGroup: String(row.workloadGroup),
                    aspgrp: String(row.aspgrp),
                    ddmConversation: String(row.ddmConversation)
                }));
```